stubs/php-file-get-contents/README.md

```diff
@@ -0,0 +1,39 @@
+# How to run
+
+```console
+./run.php expired.badssl.com 443
+```
+
+# Dependencies
+
+ * php
```

Comment:
Maybe mention in the Deps that _PHP (version >= 4.3.0) [compiled with OpenSSL support](http://php.net/manual/en/openssl.installation.php)_. E.g. the Dockerfile doesn't have any `php_openssl? package per se? Version dep requirement comes from the: https://secure.php.net/manual/en/function.file-get-contents.php